src/Directory.Build.props

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup>
```

Comment:
The properties that I had in here were intentionally focused specifically on NuGet-related things that could be pulled into a separate package and included into every package. If you want to have extra stuff in here, it would be nice to at least put it into a separate PropertyGroup to make this easier to migrate.